### ISSUE TITLE
Basic Authentication via username password 

### ISSUE THREAD

**Issue by ericpassmore** [eosnetworkfoundation/replay-test#34]
Depends on Issue #20 Nginx setup. Will be configured in Nginx.

Create a basic username/password authentication 
- admin role 
- readonly role

**Comment by ericpassmore** [comment]
after consideration, this is not a good enough solution. Closing in favor of issue #44
